snap/snapcraft.yaml

```diff
@@ -117,9 +119,12 @@ parts:
       export USE_DOCKER=1
       export DEBUG=1
       case "$(uname -m)" in
-        aarch64|ppc64le|s390x)
+        aarch64)
           sudo -E PATH=$PATH make initrd DISTRO=alpine
         ;;
+        ppc64le|s390x)
+          sudo -E PATH=$PATH make image DISTRO=ubuntu
```

Comment:
I guess we already have two different images, but ooi, why Ubuntu?

Reply:
Alpine requires a musl agent (which cannot be built on ppc64le & s390x because there is no such Rust target).

Reply:
Might be worth a comment in the code or commit to remind me/us of that ;)

Reply:
updated